components/patient-data/ui/src/main/resources/PhenoTips/GeneVariantClass.xml

```diff
@@ -66,19 +66,19 @@ $!{value}
       <validationRegExp/>
       <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
     </cdna>
-    <genesymbol>
+    <gene>
```

Comment:
@buske Just a heads-up about this change in the name of the field that references the gene in GeneVarientClass, which may break some PhenomeCentral functionality (e.g. https://github.com/phenotips/patient-network/blob/cb3b482e33b70ac8ccb63ccafd937fb6b9c9cf5d/similarity-data-impl/src/main/java/org/phenotips/data/similarity/internal/DefaultPatientGenotype.java#L150 ).